app/actions/remote/entry/common.ts

```diff
@@ -161,6 +163,13 @@ export const fetchAppEntryData = async (serverUrl: string, sinceArg: number, ini
         }
     }
 
+    // Fetch data retention policies
+    let dataRetentionPolicies = {};
+    const isDataRetentionEnabled = await getIsDataRetentionEnabled(database);
+    if (isDataRetentionEnabled) {
+        dataRetentionPolicies = await fetchDataRetentionPolicy(serverUrl);
```

Comment:
the other worry I have with this is that we only fetch the data even if there are no changes in what we had the last time we fetched them.

Reply:
True, but as of now the api doesn't support `since` parameter

https://api.mattermost.com/#tag/data-retention/operation/GetTeamPoliciesForUser